sigstore-conformance/src/main/java/dev/sigstore/Main.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2022 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore;
+
+import static com.google.common.io.Files.asByteSource;
+import static dev.sigstore.encryption.certificates.Certificates.toPemString;
+
+import com.google.common.hash.Hashing;
+import dev.sigstore.oidc.client.GithubActionsOidcClient;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Main {
```

Comment:
I imcline it would be better to select a less generic name than `dev.sigstore.Main` since ther might be several  tools for that.

How about `dev.sigstore.conformance.Main`?

Reply:
Good point!

[59f0f96](https://github.com/sigstore/sigstore-java/pull/245/commits/59f0f96a2f9977d916757b1b43bf5f0c3d2f1380)